modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -615,7 +597,7 @@ use {{invokerPackage}}\ObjectSerializer;
 
             } else {
                 // for HTTP post (form)
-                $httpBody = \GuzzleHttp\Psr7\Query::build($formParams);
+                $httpBody = ObjectSerializer::buildQuery($queryParams);
```

Comment:
Should be `$formParams` instead of `$queryParams`. I broke it with a quick _find -> replace_ action in my IDE, sorry. I'm filling up new PR with a fix.